src/transport/SessionManager.h

```diff
@@ -52,8 +52,26 @@
 #include <transport/raw/PeerAddress.h>
 #include <transport/raw/Tuple.h>
 
+#if INET_CONFIG_ENABLE_TCP_ENDPOINT
+#include <transport/SessionConnectionDelegate.h>
+#endif // INET_CONFIG_ENABLE_TCP_ENDPOINT
+
 namespace chip {
 
+/*
+ * This enum indicates whether a session needs to be established over a
+ * suitable transport that meets certain payload size requirements for
+ * transmitted messages.
+ *
+ */
+enum class TransportPayloadCapability : uint8_t
+{
+    kMRPPayload,  // This payload limits the size to be bound within an IPv6
+                  // packet size of 1280 bytes.
```

Comment:
I don't know what "bound within" means.

Bu also: this is not addressing the latency issue I raised months ago.  Consumers will want to be able to express "TCP", "MRP" and "Either one", and this is not doing that.

Reply:
I think I responded to your earlier comment about this. So, if the user wants to choose a low-latency transport, they should choose plain UDP. If it is plain UDP, then 'Either One' may not capture it correctly and there is additional plumbing underneath needed for that, presumably. This was the reason I left it as-is for now. If the goal of the third option is to use any transport that has a session up, that we could do, even if MRPPayload was chosen since this enum is about specifying the size and any transport that can handle it should suffice.